src/actions.js

```diff
@@ -1,14 +1,52 @@
 import {
-  graphql, formatMutation, formatPageQueryWithCount,
+  graphql, formatMutation, formatPageQueryWithCount, graphqlWithVariables,
 } from '@openimis/fe-core';
-import { ERROR, REQUEST, SUCCESS } from './utils/action-type';
+import {
+  CLEAR, ERROR, REQUEST, SUCCESS,
+} from './utils/action-type';
 import { ACTION_TYPE, MUTATION_SERVICE } from './reducer';
 
+const BILL_FULL_PROJECTION = [
```

Comment:
I wonder if that projection could be exported/published from invoice module. @olewandowski1 . It is just the consideration of re-usability of similar projection from invoice module

Reply:
It could be, but I was not entirely sure whether we want to configure this projection or not. If not, I can re-use it from fe-invoice.

Reply:
ok, we can think about this in the nearest future.